### PR TITLE
Add radar country weight continent weight

### PR DESCRIPTION
**Pull request by edouardbe**

### CODE REVIEW

**Review by SuzanOzgur — APPROVED** [review]

File: citrixitm/resource_citrixitm_platform.go [hunk 1]
```diff
@@ -76,6 +76,24 @@ func resourceCitrixITMPlatform() *schema.Resource {
...
 							Default:  10,
 						},
+						"country_list": {
+							Type:     schema.TypeString,
+							Required: false,
+						},
+						"country_weight": {
+							Type:     schema.TypeInt,
+							Required: false,
+							Default:  0,
```
Comment (pinkahd) [inline citrixitm/resource_citrixitm_platform.go:86]: Is this correct for the weight to be 0 even if we don't have a country list?

File: citrixitm/resource_citrixitm_platform.go [hunk 2]
```diff
@@ -76,6 +76,24 @@ func resourceCitrixITMPlatform() *schema.Resource {
...
+						},
+						"continent_list": {
+							Type:     schema.TypeString,
+							Required: false,
+						},
+						"continent_weight": {
+							Type:     schema.TypeInt,
+							Required: false,
+							Default:  0,
```
Comment (pinkahd) [inline citrixitm/resource_citrixitm_platform.go:95]: Same here should this still be 0 if we don't have a continent list?

**Review by pinkahd — APPROVED** [review]
Overall looks good to me, just some question on the defaults.

File: citrixitm/resource_citrixitm_platform.go (hunk 1 above)
Comment (edouardbe) [reply citrixitm/resource_citrixitm_platform.go:86]: I tried. country list empty "" and continent list missing (null), with both country and continent weight to 0 as default, the platform is updated with empty list of countries and continents, and country and continent weights set to 0
With a worldwide weight > 0, I was able to see the probes from a radar session, so yes, it works

File: citrixitm/resource_citrixitm_platform.go (hunk 2 above)
Comment (edouardbe) [reply citrixitm/resource_citrixitm_platform.go:95]: I tried. country list empty "" and continent list missing (null), with both country and continent weight to 0 as default, the platform is updated with empty list of countries and continents, and country and continent weights set to 0
With a worldwide weight > 0, I was able to see the probes from a radar session, so yes, it works

**Comment by edouardbe** [comment]
@pinkahd , I removed the default for the country and continent weights. tested, all's fine so far.

**Review by pinkahd — APPROVED** [review]
Thanks 🙏🏼

**Pull request merged**